src/cljs/main/broadfcui/page/external_importer.cljs

```diff
@@ -71,13 +71,12 @@
         (fn [{:keys [success? get-parsed-response]}]
           (if success?
             (swap! state assoc :wdl (:descriptor (get-parsed-response)))
-            (swap! state assoc :load-error (get-parsed-response false)))))))
+            (swap! state assoc :load-error true))))))
```

Comment:
I think we should show a little bit of easy-to-access context: `status-code` and (when available) `status-text`. Something like:
```
(fn [{:keys [success? get-parsed-response status-code status-text]}]
  (if success?
    (swap! state assoc :wdl (:descriptor (get-parsed-response)))
    (swap! state assoc :load-error (str "(" status-code ") " status-text)))))))
```
And then restore the `ErrorViewer` or some other rendering of `:load-error`.

It might not mean much to everyone but some users might be able to do something with it or at least give it to us to make our support triage easier.

Reply:
That's better, I agree